service/hook/common/common.go

```diff
@@ -43,13 +44,35 @@ type TransformResultModel struct {
 	SkippedByPrDescription bool
 }
 
+// MetricsResultModel ...
+type MetricsResultModel struct {
+	Event            string    `json:""`
+	GitRef           string    `json:""`
+	Status           string    `json:""`
+	Action           string    `json:""`
+	AppSlug          string    `json:""`
+	CommitID         string    `json:""`
+	Username         string    `json:""`
+	GitProvider      string    `json:""`
+	SourceBranch     string    `json:""`
+	TargetBranch     string    `json:""`
+	PullRequestID    string    `json:""`
+	RespositoryURL   string    `json:""`
```

Comment:
You have a typo in `RespositoryURL`, should be `RepositoryURL`

Reply:
Metrics models have been updated